dev/provision.py

```diff
@@ -279,7 +279,7 @@
 
```

Comment:
good catch, i think we can also remove L276-L278

Reply:
Unfortunally not, the `OR REPLACE` syntax requires a SQL extension I believe, so I'll leave that one for now.